src/data/ACTIONS/root/BLU.ts

```diff
@@ -604,4 +607,211 @@ export const BLU = ensureActions({
+	},
+	CHELONIAN_GATE: {
+		id: 23273,
+		name: 'Chelonian Gate',
+		icon: 'https://xivapi.com//i/003000/003340.png',
+		onGcd: true,
+		castTime: 2000,
+		gcdRecast: 2500,
+		cooldown: 30000,
+		cooldownGroup: BLU_COOLDOWN_GROUP.THE_ROSE_OF_DESTRUCTION,
+	},
+	DIVINE_CATARACT: {
+		id: 23274,
+		name: 'Divine Cataract',
+		icon: 'https://xivapi.com//i/003000/003341.png',
+		onGcd: true,
+	},
+	THE_ROSE_OF_DESTRUCTION: {
+		id: 23275,
+		name: 'The Rose of Destruction',
+		icon: 'https://xivapi.com//i/003000/003342.png',
+		onGcd: true,
+		castTime: 2000,
+		gcdRecast: 2500,
+		cooldown: 30000,
+		cooldownGroup: BLU_COOLDOWN_GROUP.THE_ROSE_OF_DESTRUCTION,
+	},
+	BASIC_INSTINCT: {
+		id: 23276,
+		name: 'Basic Instinct',
+		icon: 'https://xivapi.com//i/003000/003343.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	ULTRAVIBRATION: {
+		id: 23277,
+		name: 'Ultravibration',
+		icon: 'https://xivapi.com//i/003000/003344.png',
+		onGcd: true,
+		castTime: 2000,
+		gcdRecast: 2500,
+		cooldown: 120000,
+	},
+	BLAZE: {
+		id: 23278,
+		name: 'Blaze',
+		icon: 'https://xivapi.com//i/003000/003345.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	MUSTARD_BOMB: {
+		id: 23279,
+		name: 'Mustard Bomb',
+		icon: 'https://xivapi.com//i/003000/003346.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	DRAGON_FORCE: {
+		id: 23280,
+		name: 'Dragon Force',
+		icon: 'https://xivapi.com//i/003000/003347.png',
+		onGcd: true,
+		castTime: 2000,
+		gcdRecast: 2500,
+		cooldown: 120000,
+		cooldownGroup: BLU_COOLDOWN_GROUP.MATRA_MAGIC,
+	},
+	AETHERIAL_SPARK: {
+		id: 23281,
+		name: 'Aetherial Spark',
+		icon: 'https://xivapi.com//i/003000/003348.png',
+		onGcd: true,
+		castTime: 2000,
+		statusesApplied: ['BLEEDING'],
+	},
+	HYDRO_PULL: {
+		id: 23282,
+		name: 'Hydro Pull',
+		icon: 'https://xivapi.com//i/003000/003349.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	MALEDICTION_OF_WATER: {
+		id: 23283,
+		name: 'Malediction of Water',
+		icon: 'https://xivapi.com//i/003000/003350.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	CHOCO_METEOR: {
+		id: 23284,
+		name: 'Choco Meteor',
+		icon: 'https://xivapi.com//i/003000/003351.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	MATRA_MAGIC: {
+		id: 23285,
+		name: 'Matra Magic',
+		icon: 'https://xivapi.com//i/003000/003352.png',
+		onGcd: true,
+		castTime: 2000,
+		gcdRecast: 2500,
+		cooldown: 120000,
+		cooldownGroup: BLU_COOLDOWN_GROUP.MATRA_MAGIC,
+	},
+	PERIPHERAL_SYNTHESIS: {
+		id: 23286,
+		name: 'Peripheral Synthesis',
+		icon: 'https://xivapi.com//i/003000/003353.png',
+		onGcd: true,
+		castTime: 2000,
+	},
+	BOTH_ENDS: {
+		id: 23287,
+		name: 'Both Ends',
+		icon: 'https://xivapi.com//i/003000/003354.png',
+		cooldown: 120000,
+		cooldownGroup: BLU_COOLDOWN_GROUP.NIGHTBLOOM,
+	},
+	PHANTOM_FLURRY: {
+		id: 23288,
+		name: 'Phantom Flurry',
+		icon: 'https://xivapi.com//i/003000/003355.png',
+		gcdRecast: 2500,
+		cooldown: 120000,
+		onGcd: true, // not actually true but makes the timeline nicer
```

Comment:
Agreed that this likely will make the report more readable instead of looking like a huge gap.